sentry-core/src/main/java/io/sentry/core/SendCachedEvent.java

```diff
@@ -90,10 +103,18 @@ protected boolean isRelevantFileName(String fileName) {
 
   private void safeDelete(File file, String errorMessageSuffix) {
     try {
-      file.delete();
+      if (!file.delete()) {
+        logger.log(
+            SentryLevel.ERROR,
+            "Failed to delete '%s' " + errorMessageSuffix,
+            file.getAbsolutePath());
+      }
     } catch (Exception e) {
       logger.log(
-          SentryLevel.ERROR, "Failed to delete '%s' " + errorMessageSuffix, file.getName(), e);
+          SentryLevel.ERROR,
+          "Failed to delete '%s' " + errorMessageSuffix,
+          file.getAbsolutePath(),
+          e);
```

Comment:
H: This should be the first arg, to use this overload:

https://github.com/getsentry/sentry-android/blob/master/sentry-core/src/main/java/io/sentry/core/ILogger.java#L32